Do not crash if remote file does not exist (#620)
ansible/pylibssh: docs/changelog-fragments/620.bugfix.rst

Reviewer: Could you also add a symlink named `325.bugfix.rst` pointing to this file? This will link the same entry to both issues/PRs.

Author: Done. Also for the 208.

Reviewer: Wonderful! Thanks!

Reviewer: FTR, here's the changelog preview: https://ansible-pylibssh--620.org.readthedocs.build/en/620/changelog/#bug-fixes.

> Fix SCP transfer crashing with non-existing remote file -- by :user:`Jakuje`

Reviewer: Could you use the past tense or a non-imperative present simple with the reference to what's changed since the previous release?

Author: Something like this works?
```
Fixed crash when downloading non-existent remote file using SCP -- by :user:`Jakuje`
```
(avoiding push to let the CI run)

> Fixed crash when downloading non-existent remote file using SCP -- by :user:`Jakuje`

Reviewer: This would work. I think that it could also be something like
```suggestion
Downloading non-existent remote files via SCP no longer crashes the program -- by :user:`Jakuje`
```

Author: Updated as suggested.

> Downloading non-existent remote files via SCP no longer crashes the program. -- by :user:`Jakuje`

Reviewer: I suppose I forgot to clarify what counts as the end of a sentence :)
```suggestion
Downloading non-existent remote files via SCP no longer crashes the program -- by :user:`Jakuje`.
```
